django/thunderstore/community/tasks.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from typing import Optional
+
+from celery import shared_task
+
+from thunderstore.community.models import Community, CommunityAggregatedFields
+from thunderstore.core.settings import CeleryQueues
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(queue=CeleryQueues.BackgroundTask)
+def update_community_aggregated_fields(batch_size: Optional[int] = None) -> None:
```

Comment:
I'd also yeet the `batch_size` argument since we don't use it but that's hard to know 1 year from now, making the task seem more complex than it is. As far as I'm concerned it can be removed from the aggregation methods too given it's probably not going to see use anyway.

Reply:
Removed.